src/test/java/com/desiato/puresynth/BaseTest.java

```diff
@@ -0,0 +1,35 @@
+package com.desiato.puresynth;
+
+import com.desiato.puresynth.repositories.UserRepository;
+import com.desiato.puresynth.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class BaseTest {
```

Comment:
You should extend existing test classes with this base class. Adding dead code is usually a bad practice because the code might stay orphaned.